jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/config/KeycloakConfigurationEndpoint.java

```diff
@@ -59,11 +59,13 @@ public Response configurationFile() throws JsonProcessingException {
 
         private String realm = "aerogear";
         @JsonProperty("auth-server-url")
-        private String authServerUrl = "/auth";
```

Comment:
we can still keep the KC defaults here, since that internal/nested cfg is basically a KC config

Reply:
but that's ok if not :)